docs/source/gbd2019_models/causes/diarrheal_diseases/index.rst

```diff
@@ -219,7 +219,7 @@ Data Description
 	* - i
 	  - S
 	  - I
-	  - :math:`\frac{\text{incidence_rate_c302}}{1-\text{incidence_rate_c302}*(\text{duration_c302} / 365)`
+	  - :math:`\frac{\text{incidence_rate_c302}}{1-\text{incidence_rate_c302}*(\text{duration_c302} / 365)}`
```

Comment:
Is there any change I need to make related to this?

Reply:
No, this was just a formatting bug that I fixed